di-ipv-credential-issuer-stub/src/main/resources/templates/authorize.mustache

```diff
@@ -208,5 +216,27 @@
 <script>
     window.GOVUKFrontend.initAll();
 </script>
+<script>
+    var dropdownData = {{{cri_stub_data}}}
+    var filtered = dropdownData.filter(x => x.criType === "{{cri-name}}")
+    $(document).ready(function() {
+        var select = $('<select class="govuk-select" id="test_data" name="sort">')
+        select.append($("<option selected disabled hidden value=''>").text('Select from dropdown...'))
+        $(filtered).each(function() {
+            select.append($("<option>")
+            .prop('value', this.label)
+            .text(this.label));
+        });
+
+        var label = $('<label class="govuk-label" for="sort">').text("Select CRI stub data: ");
+        $('#test_data_block').append(label).append(select);
+        $("#test_data").on("change", function(event) {
+            var val = $(event.target).val();
+            var {payload} = filtered.find(x => x.label === val)
+            $("#jsonPayload").val(JSON.stringify(payload, undefined, 4))
+        });
+    });
+</script>
```

Comment:
Would this be better as a separate JS file, rather than inlined like this?

Reply:
From a slack chat "For the small amount of javascript it will be fine, otherwise we will have to deal with cache busting etc if we had a separate js file"